README.md

```diff
@@ -9,11 +9,13 @@ C API to Qt for Qlift
 - `ENABLE_SHARED`: Enable or disable building qlift-c-api as a shared library. Default is ON.
 - `ENABLE_STATIC`: Enable or disable building qlift-c-api as a static library. Default is OFF.
 
-To install as a shared library in ```/usr/local```:
+To install as a shared library in ```/usr/local``` (default install directory):
 ```sh
 git clone https://github.com/Longhanks/qlift-c-api
 cd qlift-c-api
-cmake CMAKE_INSTALL_PREFIX=/usr/local .
+mkdir build
+cd build
+cmake ..
 make
 sudo make install # root access may not be nessasary if you have /usr/local set up right
```

Comment:
If you don't mind, yes, I'd expect users to know how to run `make install` with apprpriate permissions. Changing ownership of 'system' directories isn't something I'd recommend.